server/lib/security/fraud.ts

```diff
@@ -29,6 +29,7 @@ export const getUserStats = async (user: typeof models.User, interval?: string):
     `
     ${BASE_STATS_QUERY} 
     WHERE o."CreatedByUserId" = :userId
+    AND o."deletedAt" IS NULL
```

Comment:
This one was also [adding](https://sentry.io/organizations/open-collective/performance/oc-api:f50babda231f4d3d9fa695b0c46b8e17/?project=5199682&query=transaction.duration%3A%3C15m+http.method%3APOST&statsPeriod=24h&transaction=GraphQL%3A+CreateOrder#span-a79f42a4df93e7af) around 800ms to the `createOrder`.